### ISSUE TITLE
Updating the pip version

### ISSUE THREAD

**Issue by mr-devs** [osome-iu/osometweet#37]
@yangkcatiu, per your comment [here](https://github.com/osome-iu/osometweet/pull/35#issuecomment-764962029) about updating the pip version I wanted to make a suggestion. 

**While we have pretty thorough documentation, I'd like to build a couple of basic sample scripts for each of the methods/endpoints (or at least the central ones) prior to this update.** 

My thinking here is three-fold:
1. Documentation only gets people so far, I think that providing skeleton use-case scripts will make it much easier for people to adopt the package.
2. I think that going through this process might help to highlight other bugs or "nice-to-haves" that we didn't think about without having to build something real.
3. These scripts will inherently be valuable down the road for ourselves (as well as the lab), so we might as well try and make this part of the standard workflow. 

I am thinking something like what I started in the [`examples/`](https://github.com/osome-iu/osometweet/tree/master/examples) folder but perhaps more barebones versions to start. I can take the lead on making the "simple" versions once everything else is done.

I think we'd need three files:
- [ ] `tweet_lookup`
- [ ] `user_lookup` (with code for both `user_lookup_ids` and `user_lookup_usernames` - seems like we could do this by just commenting out a couple of lines for one endpoint versus the other and leaving notes in the comments)
    * Another approach would be to just copy the file and make individual files for each method. Both are simple and easy.
- [ ] `followers_lookup` (same as above, for both `get_followers` and `get_following`)

I am happy to take the lead on this but please feel free to claim one of these scripts if you're interested. 😄 

I will start by claiming the `user_lookup` scripts that are in there already and update them sometime this week.

**Comment by yangkcatiu** [comment]
Good idea! I can take one or two, but maybe after you finish one case so I can see the standard.

**Comment by mr-devs** [comment]
Awesome, thanks! I'll post updates on this issue.

**Comment by yangkcatiu** [comment]
Tweet look example at #47

**Comment by mr-devs** [comment]
Update...

- Streaming examples (filtered and sample) will be included in #75 
- Timeline examples (tweets and mentions) will be included in #82
